docs/fundamentals/code-analysis/style-rules/language-rules.md

```diff
@@ -38,17 +38,18 @@ Options for language rules can be specified in an EditorConfig file with the fol
 The style rules in this section are applicable to both C# and Visual Basic.
 
 - ['this.' and 'Me.' qualifiers](ide0003-ide0009.md)
-  - dotnet\_style\_qualification\_for_field
-  - dotnet\_style\_qualification\_for_property
-  - dotnet\_style\_qualification\_for_method
-  - dotnet\_style\_qualification\_for_event
+  - [dotnet\_style\_qualification\_for_field](ide0003-ide0009.md#dotnet\_style\_qualification\_for_field)
```

Comment:
I think it would be good to add these indexes to the Options H2 section in the actual ide0003-ide0009.md file too.

Reply:
Yep, I am going to add these.